packages/integration-sdk-core/src/types/step.ts

```diff
@@ -240,3 +240,10 @@ export type StepMetadata = StepGraphObjectMetadataProperties & {
    */
   ingestionSourceId?: string;
 };
+
+export type StepWrapperFunction<
```

Comment:
Just thinking out loud - this is meant to wrap the execution handler (or step execution handler) - just proposing a few alternative interface names, in case it improves clarity:

- `StepWrapperFunction`
- `ExecutionHandlerWrapperFunction`
- `StepExecutionHandlerWrapperFunction`

Reply:
Agreed